tecton_client/_internal/data_types.py

```diff
@@ -1,6 +1,8 @@
 from dataclasses import dataclass
 from typing import Any, Dict, List, Optional
 
+_SPECIAL_NUMBER_STRINGS = {"Infinity", "-Infinity", "NaN"}
```

Comment:
These are valid responses we receive? Seem odd for an int64 right?

Reply:
Yeah I tested later they actually _aren't_ valid; it should only  only for Float64. Will remove.